package.json

```diff
@@ -12,7 +12,7 @@
     "router",
     "event router"
   ],
-  "main": "dist/esp.js",
+  "main": "index.js",
```

Comment:
The problem with this, and it brings back a reason why the dist files are checked in now:
if you're using ES5 and not babel this won't work. 

Another point is that you'd not transpile stuff in your node modules folder, you'd just want to consume dependencies as UMDs.